File: python/src/tokenizer.py
if __name__ == "__main__":
    import os

    script_dir = os.path.dirname(os.path.abspath(__file__))
    data_file = os.path.join(script_dir, "..", "..", "data", "the-verdict.txt")

    with open(data_file, "r", encoding="utf-8") as f:
        raw_text = f.read()
    print("Total number of character:", len(raw_text))
    print(raw_text[:100])

File: src/tokenizer.rs
#[cfg(test)]
mod tests {
    #[test]
    fn test_reader() {
        let raw_text = std::fs::read_to_string("./data/the-verdict.txt").unwrap();
        println!("Total number of character: {}", raw_text.len());
        println!("{:?}", &raw_text[..100]);
    }
}
